plugins/plugin-site/src/templates/plugin.jsx

```diff
@@ -181,21 +184,19 @@ function PluginPage({data: {jenkinsPlugin: plugin, reverseDependencies: reverseD
                         <h5>Maintainers</h5>
                         <PluginDevelopers developers={plugin.developers} />
                     </div>
-                    {shouldShowWikiUrl(plugin.wiki) &&
+                    {shouldShowWikiUrl(plugin.wiki || {}) &&
```

Comment:
I'm going to change it so shouldShowWikiUrl takes in a string, and then we can do plugin?.wiki?.url